### PR TITLE
Create README.md on clear-intermediate-macros

### PR DESCRIPTION
**Pull request by gbetegon88**

### CODE REVIEW

File: clear-intermediate-datasets/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Clear Intermediate Datasets Macro
+
```
Comment (JaneBellaiche) [inline clear-intermediate-datasets/README.md:2]: I would add a brief description of the macro ( just 1- 2 lines)

File: clear-intermediate-datasets/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## CAUTION
+The following macro clears intermediate datasets, which can result in the irreversible loss of data.
+Always run this macro in dry-run mode first (as enabled by default) and check in the output list to 
+confirm that no dataset is unintendedly cleared.
+
+Partitioned and Shared datasts are paticularly sensitive, and specific options has been added to remove
```
Comment (JaneBellaiche) [inline clear-intermediate-datasets/README.md:8]:
```suggestion
###Partitioned and shared datasets
Partitioned and shared datasets are particularly sensitive. You can select the available options to keep your partitioned and shared datasets in the flow. 
```

File: clear-intermediate-datasets/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+them from the list of datasets to be cleared, if desired so.
+
+Be mindful of dependencies as well. Any of the follwing can be affected when clearing a dataset that contains:
+- Charts
+- Insights published in a dashboard
+- Insights shared in a workspace,
+- It's explicitily linked to an scenario step
```
Comment (JaneBellaiche) [inline clear-intermediate-datasets/README.md:15]: It is preferred to use a neutral ton than imperative verbs, for e.g 
```suggestion
### Dependencies
Any of the following datas associated to a dataset can be affected when clearing it:
- Charts will be cleared
- Insights published in a dashboard will be cleared
- Insights shared in a workspace will be cleared
-  Scenario with the dataset as input of a scenario step will break 
```

**Review by JaneBellaiche — CHANGES_REQUESTED** [review]
Just suggested a few changes on the text. Also, have you check with the macro the behavior of the different cases we quote on the "dependencies" section? Let's ensure this is all true before merging

**Comment by gbetegon88** [comment]
> Just suggested a few changes on the text. Also, have you check with the macro the behavior of the different cases we quote on the "dependencies" section? Let's ensure this is all true before merging

Thanks for your comments @JaneBellaiche. I'm planning to go over these either Thursday or Friday.

**Comment by gbetegon88** [comment]
@JaneBellaiche I readded the README.md. I think all comments should be covered now in commit: [c413a3281a5dc546ca2c426a0726efd8f26e50c3](https://github.com/gbetegon88/dataiku-contrib/commit/c413a3281a5dc546ca2c426a0726efd8f26e50c3)

File: clear-intermediate-datasets/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Clear Intermediate Datasets Macro
+
+## Description
+Save disk space by clearing intermediate datasets in the flow.
+
+## CAUTION!
```
Comment (JaneBellaiche) [inline clear-intermediate-datasets/README.md:6]: I'd remove the scary "!" 😃

File: clear-intermediate-datasets/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+## Description
+Save disk space by clearing intermediate datasets in the flow.
+
+## CAUTION
+The following macro clears intermediate datasets, which can result in the irreversible loss of data.
+Always run this macro in dry-run mode first (as enabled by default) and check in the output list to 
+confirm that no dataset is unintendedly cleared.
+
+### Partitioned and shared datasets
+Partitioned and shared datasets are particularly sensitive. Selecting the available options _Keep partitioned datasets_ and _Keep shared datasets_ will ensured that they are not cleared.
```
Comment (JaneBellaiche) [inline clear-intermediate-datasets/README.md:12]:
```suggestion
Partitioned and shared datasets are particularly sensitive. Selecting the available options _Keep partitioned datasets_ and _Keep shared datasets_ will ensure that they are not cleared. 
```

**Review by JaneBellaiche — CHANGES_REQUESTED** [review]
Just a last typo and then you can merge this PR :)

**Pull request merged**